react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+export default class ToogleSwitch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { switchOn: false };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+
+    this.setState({ switchOn: true });
+
+  }
```

Comment:
🤔 if you click again it should go back, but this only switches 1 time

Reply:
<img width="430" alt="Screen Shot 2022-08-31 at 7 59 01 PM" src="https://user-images.githubusercontent.com/106217355/187822676-15d5fa1b-686c-4b00-90ef-2547b881bfcf.png">